dataAccessObjects.py

```diff
@@ -142,6 +142,9 @@ def getFileInfo(self,imgId):
         else:
             #TODO raise exception
             return None
+    def addLikeAccount(self):
+        sql = 'UPDATE image_data SET img_like_count=img_like_count+1 WHERE img_id="{target_id}"'.format(target_id=str(target_id))
```

Comment:
The variable name should use `lower camel case`
And you hasn't declared the `target_id` above this line.